content/rest/types.textile

```diff
@@ -36,22 +36,24 @@ h2. REST Data types
 h3(#ably-exception).
   java:  io.ably.lib.types.AblyException
   ruby:  Ably::Exceptions::BaseAblyException
+  python: AblyException
   php:   Ably\Exceptions\AblyException
 
 blang[java,php].
   <%= partial 'types/_ably_exception', indent: 2, skip_first_indent: true %>
 
-blang[ruby].
+blang[ruby,python].
   <%= partial 'types/_base_ably_exception', indent: 2, skip_first_indent: true %>
 
-h3(#error-info).
-  default: ErrorInfo
-  objc,swift: ARTErrorInfo
-  java:    io.ably.lib.types.ErrorInfo
-  ruby:    Ably::Models::ErrorInfo
-  php:     Ably\Models\ErrorInfo
+blang[jsall,ruby,php,java,objc,swift].
+  h3(#error-info).
+    default: ErrorInfo
+    objc,swift: ARTErrorInfo
+    java:    io.ably.lib.types.ErrorInfo
+    ruby:    Ably::Models::ErrorInfo
+    php:     Ably\Models\ErrorInfo
```

Comment:
Nope - afaict any error response returned by realtime is just raise as an AblyException immediately https://github.com/ably/ably-python/blob/master/ably/util/exceptions.py#L35-L70. Presumably that's a pythonic way of doing it, shrug